packages/openneuro-components/src/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import './dropdown.scss'
+
+export interface DropdownProps {
+  label: Record<string, any>
```

Comment:
What is the `Record` type and where is it coming from?

Reply:
Hmm. not sure what you mean. Nell mentioned using Record over object when a prop could be a string or an element.

Reply:
It's a utility type for general key/value type mappings. https://www.typescriptlang.org/docs/handbook/utility-types.html